ZackHolmberg/COMP4350-Project
# 48 initialize wallet service

Initialized the wallet service.

how to test:

local: navigate to the wallet service and run "python -m pytest" to run the test.
docker: spin up the containers using "docker-compose up" and go to "localhost/wallet". this should show you a welcome message.

Note: The endpoints won't work on docker containers as they depend on the blockchain endpoints
